### DISCUSSION TITLE
Panopticon features and cron jobs

### DISCUSSION THREAD

**Discussion by david-unwin** [akeeba/panopticon#317] [General]
First of all, big thank you to Nicholas and the rest of the team at Akeeba for a great product. Although we only have 2 Joomla sites, this product will make it very much easier for us to manage our backups and security. 
Just a couple of questions.
1. We also use Akeeba's Ticket System (ATS). Any plans to allow us to manage ATS and cron jobs from Panopticon as well
2. And you can guess my second question!! Any plans to use Panopticon for general Cron jobs as well? The reason I ask is that using Panopticon for our Akeeba Backup jobs is much easier to set up and maintain than using Crontab.
Fully understand if Question 2 is out of the question....but thought it was worth asking

**Comment by nikosdion** [comment]
Regarding ATS, what constitutes management of it is very open-ended and depends on the context. The only thing you could get is an overview of the number of tickets by status, further broken down by private, public, and total. Basically, the little table you get when you visit ATS. This is not very useful, and it's also not very accurate since we can only refresh that information every 15' for performance reasons. It makes more sense for you to use the already existing login links for the site's backend, or have a bookmark to your ATS "Latest Open Tickets" page in your site's frontend.

Regarding generic CRON jobs, Joomla! itself already has Scheduled Tasks. You only need to set up one CRON job on your Joomla! site to run the task scheduler once every minute. Then, you can go to System, Scheduled Tasks and configure whatever you need it to do. Having Panopticon trigger Scheduled Tasks would be more confusing. Counter-intuitively, you'd need to set up one Panopticon CRON job for every site you have an integration with Scheduled Tasks, on top of your regular Panopticon CRON jobs. Otherwise, Panopticon would neither have enough time to execute its own tasks, nor the capacity to trigger Joomla's Scheduled Tasks every minute. But, if you are going to do that, what's the point? You can just as easily set up CRON jobs on your site directly. You see what I mean?

**Reply by david-unwin** [reply]
Thanks Nicholas, Yes I can see the logic of what you have said.... Thanks for pointing out. Kind regards
